src/magic_folder/test/test_config.py

```diff
@@ -323,25 +328,23 @@ class StoreLocalSnapshotTests(SyncTestCase):
     """
     def setUp(self):
         super(StoreLocalSnapshotTests, self).setUp()
-        self.alice = create_local_author("alice")
+        self.author = create_local_author("alice")
 
     def setup_example(self):
         self.temp = FilePath(self.mktemp())
-        self.global_db = create_global_configuration(
-            self.temp.child(b"global-db"),
-            u"tcp:12345",
-            self.temp.child(b"tahoe-node"),
-            u"tcp:localhost:12345",
-        )
+        self.stash = self.temp.child("stash")
+        self.stash.makedirs()
```

Comment:
This is redundant (I think) since `initialize` will do atomic_makedirs on the stash-path?

Reply:
The `atomic_makedirs` calls got left behind in `create_magic_folder` because they are trying to encapsulate both the `MagicFolderConfig` initialization _and_ the global database configuration changes.  Coordinating the three difference pieces of state (filesystem, global db, per-folder db) is definitely awkward :/  Seems like there should be a better way to manage it but I don't know what that is yet.